toucan_connectors/__init__.py

```diff
@@ -30,6 +30,8 @@
     from .mysql.mysql_connector import MySQLConnector
 with suppress(ImportError):
     from .oracle_sql.oracle_sql_connector import OracleSQLConnector
+with suppress(ImportError):
+    from .odata.odata_connector import ODataConnector
```

Comment:
Nitpicking here, sorry… but it looks like imports in `toucan_connectors/__init__` are sorted alphabetically. `odata` should therefore probably stand before `oracle`

Reply:
fixed